### ISSUE TITLE
example project

### ISSUE THREAD

**Issue by PRCV1** [PRCV1/pocketbase-csharp-sdk#24]
using Blazor WASM

**Comment by neozhu** [comment]
I'm planning to work on a small project using Blazor wasm + pocketbase and came across your project. It's fantastic! However, I had a question when running the Example. Why aren't the HTTP requests for querying and saving data including the token obtained from logging in? Usually, every request to fetch or manipulate data requires a token or cookie.


<img width="1343" alt="image" src="https://github.com/PRCV1/pocketbase-csharp-sdk/assets/1549611/9ee0df25-946e-44c4-9cfa-66493d2f224a">

**Comment by neozhu** [comment]
Haha, my mistake, it's in the CreateRequest method. 😁

**Comment by PRCV1** [comment]
yes, the CreateRequest method contains all the logic needed prior to sending an request (authentication etc)
